docs/Configuring.md

```diff
@@ -333,6 +333,7 @@ When running a job on the Config (Shared Filesystem) backend, Cromwell provides
               # "file" will compute an md5 hash of the file content.
               # "path" will compute an md5 hash of the file path. This strategy will only be effective if the duplication-strategy (above) is set to "soft-link",
               # in order to allow for the original file path to be hashed.
+              # "path+modtime" will compute an md5 hash of the file path and the last modified time. The same conditions as for "path" apply here.
```

Comment:
Unfortunately, I can't add a comment to line 332, but could you add this to the `# possible values: file, path` list too